client/src/app/views/login.js

```diff
@@ -0,0 +1,32 @@
+window.LoginView = Backbone.View.extend({
+  initialize: function() {
+    this.render();
+  },
+  events: {
+    'submit' : 'loginUser'
+  },
+  render: function() {
+    $(this.el).html(this.template());
+    return this;
+  },
+
+  loginUser: function(e){
+    e && e.preventDefault();
+    var $formUsername = this.$el.find('form .username');
```

Comment:
why memoize the form field but not the value, if the value is all that is used throughout the function?